test/components/core_spec.js

```diff
@@ -75,11 +75,13 @@ describe('Core', function() {
 
         it('adds a class "fullscreen" to core element', () => {
           const spy = sinon.spy(this.core.$el, 'addClass')
+          sinon.stub(this.core, 'isFullscreen').returns(false)
+
           expect(spy).not.to.have.been.called
 
           this.core.toggleFullscreen()
-
           expect(spy).to.have.been.calledWith('fullscreen')
+
```

Comment:
There are a number of those empty lines at the end of blocks, please remove those.